extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -471,13 +476,37 @@ export class AmpStoryPage extends AMP.BaseElement {
   }
 
   /**
-   * Finds embedded components in page and prepares them for their expanded view
-   * animation.
+   * Finds embedded components in page and prepares them.
    * @param {boolean=} forceResize
    * @private
    */
   findAndPrepareEmbeddedComponents_(forceResize = false) {
-    scopedQuerySelectorAll(this.element, EMBEDDED_COMPONENTS_SELECTORS)
+    this.addClickShieldToEmbeds_();
+    this.resizeInteractiveEmbeddedComponents_(forceResize);
+  }
+
+  /**
+   * Adds a pseudo element on top of the embed to block clicks from going into
+   * the iframe.
+   * @private
+   */
+  addClickShieldToEmbeds_() {
+    this.mutateElement(() => {
+      scopedQuerySelectorAll(this.element, EMBEDDED_COMPONENTS_SELECTORS)
```

Comment:
Sorry to be picky about this, but can we query the DOM before the `mutateElement`, and add the `...embeddedComponents_` suffix to the method name to be consistent?

I was thinking something like
```
const componentEls = scopedQuerySelectorAll();
this.mutateElement(() => {
  componentEls.forEach(...);
});
```

Doesn't matter so much here, but it's a good practice to optimize the JS execution order :)